backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -1649,25 +1822,81 @@ describe('UserResolver', () => {
             })
           })
 
+          describe('to not allowed role', () => {
+            it('throws an error', async () => {
+              jest.clearAllMocks()
+              await expect(
+                mutate({
+                  mutation: setUserRole,
+                  variables: { userId: user.id, role: 'unknown rolename' },
+                }),
+              ).resolves.toEqual(
+                expect.objectContaining({
+                  errors: [new GraphQLError('Not allowed to set user role=')],
```

Comment:
This message will change, when handling validation by schema

Reply:
i don't understand what to do here.

Reply:
My point is:

Your code:
```
@Arg('role', () => String, { nullable: true })
    role: string | null | undefined,
```

Please use schema for validation.

```
    @Arg('role', () => ROLE_NAMES, { nullable: true })
role: ROLE_NAMES | null | undefined,
```

Therefore this message changes.